common/azure-common/src/test/java/com/azure/common/implementation/RestProxyTests.java

```diff
@@ -1519,7 +1519,7 @@ public void SegmentUploadTest() throws Exception {
 
     @Host("{url}")
     interface Service22 {
-        @GET("{container}/{blob}")
```

Comment:
This is an obvious fix as there's no method parameter substituting `{container}` and `{blob}`. The mock client is also apparently expecting simply `/bytes/{int}`.